### PR TITLE
update-concepts

### PR DESCRIPTION
**Pull request by vpavicic**
[x] went through concept pages and made language changes

### CODE REVIEW

File: docs/concepts/indexing.md [hunk 1]
```diff
@@ -10,92 +10,86 @@ A database index is a data structure used to improve the speed of data retrieval
...
-and index based retrieval might not be used. Nevertheless, we are continually
-improving our index usage recognition algorithms.
-
-Information about available indexes can be retrieved by using the following
-syntax:
+Information about available indexes can be retrieved using the following syntax:
 
 ```cypher
 SHOW INDEX INFO;
 ```
```
Comment (kgolubic) [inline docs/concepts/indexing.md:80]: Add one empty line after this line.

File: docs/concepts/indexing.md [hunk 2]
```diff
@@ -10,92 +10,86 @@ A database index is a data structure used to improve the speed of data retrieval
...
-The results of this query will be all of the labels and label-property pairs
-that Memgraph currently indexes.
+This query will return all the labels and label-property pairs that Memgraph
+currently indexes.
 
-Created indexes can also be deleted by using the following syntax:
+You can delete created indexes by using the following syntax:
 ```cypher
```
Comment (kgolubic) [inline docs/concepts/indexing.md:85]: Add one empty line before this line.

File: docs/concepts/storage.md [hunk 3]
```diff
@@ -4,120 +4,118 @@ title: Storage
...
 
 Snapshots are taken periodically during the entire runtime of Memgraph. When
 a snapshot is triggered, the whole data storage is written to disk. The
 snapshot file provides a quicker way to restore the database state.
 
-Database recovery is done on startup from the most recently found snapshot
-file. Since the snapshot may be older than the most recent update logged in
-the WAL file, the recovery process will apply the remaining state changes
-found in the said WAL file.
+You can also generate a snapshot file by using Memgraph's import tools. For more
```
Comment (kgolubic) [inline docs/concepts/storage.md:22]: I'm not sure if snapshots generated by import tools fall into this category. Can you please double-check this?

File: docs/concepts/storage.md [hunk 4]
```diff
@@ -4,120 +4,118 @@ title: Storage
...
-Although openCypher supports properties on edges, they are often not used. If
-you have a use-case that doesn't use properties on edges you can specify a flag
-in the Memgraph configuration file which will disable the usage of properties
-on edges.
+If you have a use-case that doesn't use properties on relationships, you can
+specify a flag in the Memgraph configuration file to disable them and reduce
+memory usage.
 ```
 --storage-properties-on-edges=false
 ```
```
Comment (kgolubic) [inline docs/concepts/storage.md:104]: Add an empty line before this one.

File: docs/concepts/storage.md [hunk 5]
```diff
@@ -4,120 +4,118 @@ title: Storage
...
+### Disabling properties on relationships
 
-Although openCypher supports properties on edges, they are often not used. If
-you have a use-case that doesn't use properties on edges you can specify a flag
-in the Memgraph configuration file which will disable the usage of properties
-on edges.
+If you have a use-case that doesn't use properties on relationships, you can
+specify a flag in the Memgraph configuration file to disable them and reduce
+memory usage.
 ```
```
Comment (kgolubic) [inline docs/concepts/storage.md:102]: Add one empty line before this line.

File: docs/concepts/storage.md (hunk 4 above)
Comment (kgolubic) [inline docs/concepts/storage.md:104]: Add one empty line after this line.

**Review by kgolubic — CHANGES_REQUESTED** [review]
I've left a few comments. Most of them are related to formatting issues.

File: docs/concepts/graph-algorithms.md [hunk 6]
```diff
@@ -6,52 +6,52 @@ sidebar_label: Graph algorithms
...
+those objects as abstractions named `nodes` (also called `vertices`) and the
+relations between them as an abstractions named `relationships` (also called
+`edges`).
+
+It turns out that a lot of real-world problems can be successfully modeled using
+graphs. Some natural examples would contain railway networks between cities,
+computer networks, piping systems and Memgraph itself.
+
+This article outlines some of the most important graph algorithms that are
+internally used by Memgraph. We believe that advanced users could significantly
```
[the remark this answers is not in the document]
Comment (vpavicic) [reply docs/concepts/graph-algorithms.md:20]: Maybe advanced users are still not using them as much as they could... I'm not sure what is implied... will ask in the channel

File: docs/concepts/graph-algorithms.md [hunk 7]
```diff
@@ -17,19 +17,22 @@ graphs. Some natural examples would contain railway networks between cities,
...
 effectively and efficiently.
 
-Contents of this article include:
+This article outlines:
 
   * [Breadth-first search (BFS)](#breadth-first-search)
   * [Weighted shortest path (WSP)](#weighted-shortest-path)
 
+Memgraph has a lot more graph algorithms to offer besides these two and they are
+all a part of [MAGE](/mage/overview.md).
```
Comment (kgolubic) [inline docs/concepts/graph-algorithms.md:33]: URL for MAGE needs to be fixed.

**Review by kgolubic — APPROVED** [review]

**Pull request merged**